eZ/Publish/Core/Persistence/Cache/ContentTypeHandler.php

```diff
@@ -349,8 +365,28 @@ public function publish( $typeId )
         // Clear type cache and all identifier cache (as we don't know the identifier)
         $this->cache->clear( 'contentType', $typeId );
         $this->cache->clear( 'contentType', 'identifier' );
+        $this->cache->clear( 'searchableFieldMap' );
 
         // clear content cache
         $this->cache->clear( 'content' );//TIMBER! (possible content changes)
     }
+
+    /**
+     * @see \eZ\Publish\SPI\Persistence\Content\Type\Handler::getFieldMap
```

Comment:
Fixed in https://github.com/ezsystems/ezpublish-kernel/commit/ea68a7308fbb2d49301821e454850b915d7c7bf0